src/main/java/org/jilt/internal/AbstractBuilderGenerator.java

```diff
@@ -331,12 +331,18 @@ protected final TypeVariableName mangleTypeParameter(TypeVariableName typeVariab
     }
 
     private ParameterSpec setterParameterSpec(VariableElement attribute, TypeName parameterType) {
-        ParameterSpec.Builder ret = ParameterSpec.builder(parameterType,
-                this.attributeSimpleName(attribute));
-        AnnotationMirror nullableAnnotation = this.firstAnnotationCalledNullable(attribute);
+        ParameterSpec.Builder ret = ParameterSpec.builder(parameterType, this.attributeSimpleName(attribute));
+        AnnotationMirror nullableAnnotation = firstAnnotationCalledNullable(attribute);
         if (nullableAnnotation != null) {
             ret.addAnnotation(AnnotationSpec.get(nullableAnnotation));
         }
+
+        for (AnnotationMirror annotation : attribute.getAnnotationMirrors()) {
+            if (!annotation.equals(nullableAnnotation)) {
+                ret.addAnnotation(AnnotationSpec.get(annotation));
+            }
+        }
+
```

Comment:
I feel like we're copying too much here - what if the annotation is from a field (because `@Builder` was placed on the class level) , but it's not allowed to be placed on a parameter? We would be generating code that doesn't compile here.

Can you please add a check that makes sure the given annotation can be placed on a parameter before you copy it?